packages/docs/pages/apps/stripe/triggers.md

```diff
@@ -0,0 +1,18 @@
+---
+favicon: /favicons/stripe.svg
+items:
+  - name: Payout
+    desc: Triggers when stripe sent a payout to a third-party bank account or vice versa.
+    org: Stripe Documentation
+    orgLink: https://stripe.com/docs/api/payouts/object
+  - name: Balance Transaction
```

Comment:
```suggestion
  - name: New Balance Transactions
```

Reply:
We want to probably say "New balance transactions" though, as I think it's better to suit the Stripe terminology.